Test/allocated1/dafny0/Maps.dfy.expect

```diff
@@ -31,7 +31,7 @@ Maps.dfy(447,17): Error: assertion might not hold
 Maps.dfy(448,13): Error: assertion might not hold
 Maps.dfy(520,18): Error: assertion might not hold
 Maps.dfy(525,20): Error: assertion might not hold
-Maps.dfy(562,2): Error: decreases expression might not decrease
-Maps.dfy(576,2): Error: decreases expression might not decrease
+Maps.dfy(562,2): Error: decreases clause might not decrease
```

Comment:
Here I have a doubt, but I think this message is better, since it's not just one expression, it can be this special kind of tuple.